### ISSUE TITLE
Install fail on Windows if SWIG is in the Path

### ISSUE THREAD

**Issue by Cilyan** [Perl5-Alien/Alien-Build#166]
`cpanm Alien::m4` fails if the [SWIG ](http://www.swig.org/)folder is in the path, even if it is placed after Strawberry Perl installation folders. It installs properly if the Path variable is temporarily cleaned.

```
Alien::Build::Plugin::Core::Download> setting version based on archive to 1.4.18
Alien::Build::Plugin::Core::Download> downloaded m4-1.4.18.tar.gz
"C:\Strawberry\perl\bin\perl.exe" -MAlien::Build::MM=cmd -e build
Alien::Build::CommandSequence> + sh configure --prefix=/C/STRAWB~1/perl/site/lib/auto/share/dist/Alien-m4 --with-pic --disable-shared
configure: WARNING: unrecognized options: --with-pic, --disable-shared
configure: error: cannot find sources (src/m4.h) in /c/swigwin-4.0.1 or ..
external command failed at C:/Strawberry/perl/vendor/lib/Alien/Build/CommandSequence.pm line 87.
gmake: *** [Makefile:1005: _alien/mm/build] Error 2
-> FAIL Installing Alien::m4 failed.
```

**Comment by plicease** [comment]
I'm able to reproduce this with both `Alien::m4` and `Alien::FFI` so, if there is a fix for this it should be in `Alien-Build`.

**Comment by plicease** [comment]
Looks like this should fix it:

0b75aff5e2bf0b7c95499a25cc4cb06ca87b4a9b

Possibly MSYS bash puts `.` at the end of the PATH, and it is finding `configure` in the swig directory instead.  Thanks for reporting this.

**Comment by plicease** [comment]
Shipped to CPAN as 2.05_01.  Should be a production version soon.
